setup.py

```diff
@@ -39,7 +42,7 @@
     },
     install_requires=[
         'typer',
-        'datameta-client-lib==0.0.1',
+        f'datameta-client-lib>={major_version},<{major_version+1}',
```

Comment:
We need a stricter minimum version here. When I installed the package I still had `0.0.1` in my package cache and it wasn't upgraded - but `0.0.1` doesn't know `submissions.label` yet, hence the client crashed.

Reply:
Oh right, OK, lets fix the lower bound to 0.4.0. Once we are at >=1.0.0, we should however, take care that you can mix and match client and server with the same major version, right?

Reply:
IIRC in our previous discussion we agreed that minor API versions may add endpoints or attributes that don't break older clients. But that means in turn that newer clients may require a certain minor version if the make use of these additions.